### ISSUE TITLE
Lambda with positional-only arguments has wrong inferred type

### ISSUE THREAD

**Issue by weakish** [microsoft/pyright#575]
Lambda with positional-only arguments has wrong inferred callable type.
`/` confuses Pyright on inferred argument numbers (inferred one more argument than expected).
For example:

```python
plus_one: Callable[[int], int] = lambda x, /: x + 1
```

Pyright gives this type error:

```
Expression of type '(x: int, /) -> int' cannot be assigned to declared type '(p0: int) -> int'
  Function accepts too few parameters. Expected 2 but got 1.
```

Tested on pyright 1.1.28 (vscode extension)

**Comment by erictraut** [comment]
Thanks for your bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.29, which I just published.
